src/core/error/error.go

```diff
@@ -0,0 +1,26 @@
+package error
+
+type BadRequest struct {
+	message      string
+	statusRecord string
+	trace        string
+	error        error
```

Comment:
ここはインターフェースなので、errorだけで良いですよ。
```suggestion
	error 
```

こういうことです。

Reply:
他も同様です。